scalac-scoverage-plugin/src/main/scala/scoverage/plugin.scala

```diff
@@ -108,9 +121,20 @@ class ScoverageInstrumentationComponent(val global: Global, extraAfterPhase: Opt
   private var coverageFilter: CoverageFilter = AllCoverageFilter
 
   def setOptions(options: ScoverageOptions): Unit = {
+
     this.options = options
+
+    // If useEnvironment is true, we set the output directory to output classpath,
+    // i.e to the directory where compiler is going to output the classfiles.
+    if(options.useEnvironment){
+      settings.outputDirs.getSingleOutput match {
+        case Some(dest) => options.dataDir = dest.toString() + "/META-INF/scoverage"
```

Comment:
Can we make `META-INF/scoverage` a configurable override option that could be passed in?